### ISSUE TITLE
Billing: Add the ability to make bulk changes to purchases in WordPress.com

### ISSUE THREAD

**Issue by davipontesblog** [Automattic/wp-calypso#47806]
I had an interaction in live chat with a user who had 136 sites they manage for students, and they needed to set all of those from "auto-renewal" to "manual renewal" (turning off auto-renewal) – which is, unfortunately, not something easily done in calypso as a subscriber at this time.

It would be nice if users had a way to toggle these "on" and "off" individually from one central billing management interface, which would have avoided all of this manual work.

Nice to have: If they could select many items and then use one toggle change for "on" and "off", that would be even better.

Internal context: p7DVsv-9fp-p2#comment-32620

**Comment by davipontesblog** [comment]
@fditrapani is this something that your team would already be working on?

**Comment by fditrapani** [comment]
Thanks for the ping @davipontesblog! Yes, we have this item on our roadmap it just hasn't been planned out yet. Anything you can share about how often this comes up, or how many customers this could benefit, will help us with prioritization.

**Comment by davipontesblog** [comment]
> Anything you can share about how often this comes up, or how many customers this could benefit, will help us with prioritization.

I would say it's a very low incidence since we don't have many people that have over 100 sites on their account. But if we start marketing more towards folks who manage multiple sites, schools, etc, we may want to prioritize more.

**Comment by fditrapani** [comment]
> But if we start marketing more towards folks who manage multiple sites, schools, etc, we may want to prioritize more.

Absolutely! We'll keep an eye out for that and adjust our priorities accordingly.
